internal/services/spec/validator/validator_workdir.go

```diff
@@ -22,8 +22,7 @@ func newValidatorWorkDir(utils *utils) *validatorWorkDir {
 func (v *validatorWorkDir) Validate(doc spec.Document) []arch.Notice {
 	notices := make([]arch.Notice, 0)
 
-	rootDir := filepath.Dir(doc.Version().Reference.File)
-	absPath := fmt.Sprintf("%s/%s", rootDir, doc.WorkingDirectory().Value)
+	absPath := filepath.Join(v.utils.projectDir, doc.WorkingDirectory().Value)
```

Comment:
replaced a fmt.Sprintf to the filepath.Join because it uses right path separator for a different operating systems (windows, nix)